app/src/main/java/com/gathering/android/common/composables/SnackBarHelper.kt

```diff
@@ -0,0 +1,40 @@
+package com.gathering.android.common.composables
+
+import androidx.compose.material3.SnackbarDuration
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.SnackbarResult
+import androidx.compose.runtime.compositionLocalOf
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+val SnackBarHelpers = compositionLocalOf { SnackBarHelper() }
+
+data class SnackBarHelper(
+    val scope: CoroutineScope? = null,
+    val snackbarHostState: SnackbarHostState? = null
+) {
+    fun showSnackbar(
+        text: String,
+        actionText: String,
+        onDismissed: () -> Unit,
+        onActionPerformed: () -> Unit
+    ) {
+        scope?.launch {
```

Comment:
i am not sure if the scope is required for functionality , but if it is, we might want to handle in case it is null like this:

    scope ?: throw IllegalArgumentException("Coroutine scope must be provided")